Microsoft.Azure.Cosmos/src/Resource/Settings/DistanceFunction.cs

```diff
@@ -0,0 +1,31 @@
+﻿//------------------------------------------------------------
+// Copyright (c) Microsoft Corporation.  All rights reserved.
+//------------------------------------------------------------
+namespace Microsoft.Azure.Cosmos
+{
+    using System.Runtime.Serialization;
+
+    /// <summary>
+    /// Defines the distance function for a vector index specification in the Azure Cosmos DB service.
+    /// </summary>
+    public enum DistanceFunction
+    {
+        /// <summary>
+        /// Represents the euclidean distance function.
+        /// </summary>
+        [EnumMember(Value = "Euclidean")]
```

Comment:
Can you use `nameof` here?

```suggestion
        [EnumMember(Value = nameof(Euclidean))]
```